corehq/apps/app_manager/templates/app_manager/v2/partials/module_view_settings.html

```diff
@@ -119,17 +119,23 @@ <h4 class="panel-title panel-title-nolink">{% trans "Multimedia Settings" %}</h4
             </div>
           </div>
 
-           {% if app.advanced_app_builder %}
-           <div class="panel panel-appmanager">
-            <div class="panel-heading">
-              <h4 class="panel-title panel-title-nolink">{% trans "Advanced Settings" %}</h4>
-            </div>
-            <div class="panel-body">
-              {% include 'app_manager/v2/partials/case_list_form_setting.html' %}
+           {% if not module.is_surveys %}
+             {% if app.advanced_app_builder or module.case_list_form.form_id or module.case_list.show or module.task_list.show %}
+             <div class="panel panel-appmanager">
+              <div class="panel-heading">
+                <h4 class="panel-title panel-title-nolink">{% trans "Advanced Settings" %}</h4>
+              </div>
+              <div class="panel-body">
+                {% include 'app_manager/v2/partials/case_list_form_setting.html' %}
+
+                {% include 'app_manager/v2/partials/case_list_setting.html' with LABEL="Case List Menu Item" DESC="An item in the module's menu that lets you browse the case list without moving on to fill out a form." SLUG="case_list" case_list=module.case_list %}
```

Comment:
I don't think so...that is, I don't think there's a straightforward way to do it in the template...that is, we don't do it in other, similar places.